core/modules/layout/css/layout.flexible.admin.css

```diff
@@ -67,3 +65,10 @@
   background-clip: content-box;
   text-align: center;
 }
+
+.l-flexible-row .layout-editor-region {
+  border: 1px solid white;
+}
```

Comment:
We should remove this in order to be consistent with the dashed border around the regions in the "Manage blocks" UI (it will also help a bit with the contrast issues).

Reply:
Good idea.

Reply:
@klonos the dashed border has meaning ("you can drop things here") that doesn't apply on this interface, that's why there are no dashed borders here. This border is to prevent the dashes from appearing.